### DISCUSSION TITLE
Rendering issue with Quarto

### DISCUSSION THREAD

**Discussion by jrycw** [posit-dev/great-tables#375] [General]
I encountered an issue while working on my submission for the [2024 Table Contest](https://forum.posit.co/t/euro-ncap-safety-ratings-2023-table-contest/187584/4). When rendering the final table, the result isn't as good as the output from `GT.save()`. I tried setting `page-layout` to `full`, which helped slightly, but not much. I also tried setting `page-layout` to `custom`. While this made the table look great, it threw off the layout of the other contents.

My current workaround is to output the raw HTML via `GT.as_raw_html()` and paste it into another Quarto document to render. I'm not sure if there's anything we can adjust on our end or if I need assistance from the Quarto team. @rich-iannone and @machow , could you please share your thoughts with me?

## Page Layout: Article
![image](https://github.com/posit-dev/great-tables/assets/67060418/abd34e9c-f4c7-4d11-a450-08e1aefd7666)

## Page Layout: Full
![image](https://github.com/posit-dev/great-tables/assets/67060418/f69d85a1-908d-482d-99ad-3da608aff164)

## Page Layout: Custom
![image](https://github.com/posit-dev/great-tables/assets/67060418/40a77963-9f53-4b4e-bfaf-c06d49c60bf2)

**Comment by machow** [comment]
Does it help if you add `html-table-processing: none` to the yaml topmatter? I think w/o it quarto does some extra styling that can screw up table display :/

cf https://github.com/posit-dev/great-tables/issues/372

**Reply by jrycw** [reply]
@machow, thanks so much for the feedback. While the table styling is still slightly different from `GT.as_raw_html()`, it's much better now.
